docs/0.4/concepts/security_considerations.md

```diff
@@ -0,0 +1,164 @@
+# Security Considerations
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+Splinter provides several levels of security and customizable permissions.
+
+## Cryptographic signing with public/private keys
+
+Like Sawtooth and other distributed ledgers, Splinter uses cryptographic
+signing with public/private keys. Each user and transaction-submitting entity
+has a public key and an encrypted private key.
+
+All transactions that affect shared state are signed with the encrypted private
+key. This includes circuit creation (proposals and votes) and smart contract
+operations (uploads, configuration, and changes), as well as functions
+performed by the Splinter daemon and application-specific services.
+
+## User and credential management with Biome
+
+Biome is the Splinter component that manages the relationship between
+user-based data and Splinter's key-based user information. Put simply, Biome
+maps user IDs and passwords (or other authentication credentials) to the
+public/private keys that the rest of Splinter uses to sign transactions.
+
+Biome isolates all user-based data from the rest of Splinter, using a
+Biome-specific database to store the mapping between user data and keys. All
+other Splinter functions use only key-based signing to identify actors and
+verify that they have permission for the requested operation.
+
+It's important to note that Splinter does not encrypt private keys. The client
```

Comment:
This should be reworded -- "It is important to note that private keys stored in Biome are encrypted on the client-side; Splinter does not also encrypt it within the REST API." -- because otherwise it lacks context and sounds like a lacking feature (which its not).

Reply:
Reworded.